docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -7,6 +7,64 @@
 
 The ServiceNow ITSM connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow incidents.
 
+[float]
+[[servicenow-itsm-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow ITSM connector or <<servicenow-itsm-connector-update, updating an existing one>>:
+
+* Install Elastic for ITSM from the https://store.servicenow.com/sn_appstore_store.do#!/store/home[ServiceNow Store].
+* Create a ServiceNow integration user and assign it the appropriate roles.
+* Create a CORS rule.
+
+*Create a ServiceNow integration user*
+
+To ensure authenticated communication between Elastic and ServiceNow, create a ServiceNow integration user and assign it the appropriate roles. 
+
+. Go to *System Security -> Users*.
+. Click *New*.
+. Complete the form.
+. Assign the integration user the following roles: 
+* `import_set_loader`
+* `Import_transformer`
+* `personalize_choices`
+* `sn_si.basic`
+* `x_elas2_sir_int.integration_user`
+. Click *Submit* to create the user.
+
+*Create a CORS rule*
+
+A CORS rule is required for communication between the Elastic and ServiceNow. To create a CORS rule:
+
+. Go to *System Web Services -> CORS Rules*.
+. Click *New*.
+. Configure the rule settings to use the Elastic SIR API and the Kibana URL:
+* *REST API*: Choose `x_elas2_sir_int/elastic_api`.
```

Comment:
`x_elas2_sir_int/elastic_api` -> `x_elas2_inc_int/elastic_api`

Reply:
I think it should be better to put the name also: Choose `Elastic ITSM API [x_elas2_inc_int/elastic_api]`. This how will be shown in ServiceNow.